packages/rs-drive-abci/tests/strategy_tests/upgrade_fork_tests.rs

```diff
@@ -490,6 +502,166 @@ mod tests {
         handler.join().expect("Thread has panicked");
     }
 
+    #[test]
+    fn run_chain_on_epoch_change_with_new_version_and_removing_votes() {
+        // Add a new version to upgrade to new protocol version only with one vote
+        const TEST_PROTOCOL_VERSION_4_WITH_1_HPMN_UPGRADE: u32 =
+            (1 << TEST_PROTOCOL_VERSION_SHIFT_BYTES) + 4;
+
+        let mut test_platform_v4 = PLATFORM_V1.clone();
+        test_platform_v4
+            .drive_abci
+            .methods
+            .protocol_upgrade
+            .protocol_version_upgrade_percentage_needed = 1;
```

Comment:
Why is this part of this test?

Reply:
We agreed to move this constant to the platform version in the comments above. Since `MasternodeListChangesStrategy` can be set only on `run_chain_for_strategy` and it's falling after few more blocks the solution is to change this percentage to 1 instead of 75 so 1 vote is enough to switch to new version.